RangeBarSample/src/main/java/com/appyvet/rangebarsample/RecyclerViewActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.appyvet.rangebarsample;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+
+public class RecyclerViewActivity extends Activity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Removes title bar and sets content view
+        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
+        setContentView(R.layout.activity_recyclerview);
+        RecyclerView recyclerView = findViewById(R.id.recyclerView);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+        recyclerView.setAdapter(new Adapter());
+    }
+
+    public static class Adapter extends RecyclerView.Adapter<VH> {
+
+        @NonNull
+        @Override
+        public VH onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+            return new VH(LayoutInflater.from(viewGroup.getContext())
+                    .inflate(R.layout.activity_recyclerview_adapter, viewGroup, false));
+        }
+
+        @Override
+        public void onBindViewHolder(@NonNull VH vh, int i) {
+
+        }
+
```

Comment:
Please add the following lines so that state/progress of rangebar is kept while scrolling the recyclerview as defined [here](https://stackoverflow.com/a/43179233/1118886).

```
 @Override
        public long getItemId(int position) {
            return position;
        }

        @Override
        public int getItemViewType(int position) {
            return position;
        }
```

Reply:
Done.